static/src/javascripts/projects/common/modules/experiments/tests/contributions-header.js

```diff
@@ -43,8 +43,8 @@ define([
             }));
 
             return fastdom.write(function () {
-                var a = $('.brand-bar__item--register');
-                a.replaceWith($newThing);
+                var a = $('.brand-bar__item--profile.popup-container.has-popup.js-profile-nav.is-signed-in');
```

Comment:
So does there end up being any kind of race-condition, given that `is-signed-in` is [dynamically added]( https://github.com/guardian/frontend/blob/220c46ad/common/app/templates/inlineJS/nonBlocking/showUserName.scala.js#L15)?